### ISSUE TITLE
[med] JAWS+Chrome: Chat: The meaning of contact online status icon not communicated

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5672]
## Action Performed:
Prerequisites:
- User has created personal account and is logged into same.
- User has multiple chats in chat history.

1. Using JAWS+Chrome, open https://staging.new.expensify.com
2. Tab to focus the profile icon button or any other contacts with the green dot badge icon online status indicator and note how it is announced.

## Expected Result:
The meaning of the online status icon should be announced when the buttons are focused. "Button [Label] [online/offline/busy/etc...]".

## Actual Result:
The meaning of the online status badge icon is never announced.

HTML:
`<div class="css-1dbjc4n" style="background-color: rgb(198, 201, 202); border-radius: 20px; height: 40px; width: 40px;"><div class="css-1dbjc4n r-633pao"><div class="css-1dbjc4n r-1mlwlqe r-1udh08x r-417010" style="border-radius: 40px; height: 40px; width: 40px;"><div class="css-1dbjc4n r-1niwhzg r-vvn4in r-u6sd8q r-4gszlv r-1p0dtai r-1pi2tsx r-1d2f490 r-u8s1d r-zchlnj r-ipm5af r-13qz1uu r-1wyyakw" style="background-image: url(&quot;https://d1wpcgnaa73g0y.cloudfront.net/ef890196cc06299e234adc5ca42cf1e61e288ba7_128.jpeg&quot;
);"></div><img alt="" draggable="false" src="https://d1wpcgnaa73g0y.cloudfront.net/ef890196cc06299e234adc5ca42cf1e61e288ba7_128.jpeg"
class="css-9pa8cd"></div></div><div class="css-1dbjc4n r-u8s1d" style="align-items: center; background-color: rgb(3, 212, 124); border-color: rgb(250, 250, 250); border-radius: 6px; border-width: 2px; bottom: -1px; height: 12px; justify-content: center; right: -1px; transform: rotate(0deg) scale(1); width: 12px; z-index: 10; -webkit-box-align: center; -webkit-box-pack: center;"></div></div>`

## Workaround:
No workaround

## Area issue was found in:
Chat page

## Failed WCAG checkpoints
1.1.1

## User impact:
Screen readers users will not get the information that the status icons provide.

## Suggested resolution:
Use aria-describedby attribute to add off-screen description text to the buttons that indicate the meaning of the status icons when displayed. Alternatively, include the meaning of the status icon in the aria-label attribute of the button in order to announce it to the screen reader users.
https://www.w3.org/TR/2017/REC-wai-aria-1.1-20171214/#aria-describedby

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5187248_Online_status](https://user-images.githubusercontent.com/88733897/136197887-eacdc3b6-a1b9-465c-8a77-ce81b98cd5a8.png)

https://user-images.githubusercontent.com/88733897/136197897-59c31d14-faf3-4cbe-a447-99a0a5fa3ccd.mp4

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
